services/ocm/README.md

```diff
@@ -57,8 +127,12 @@ The data backend of the `ocminvitemanager` is configurable. The only supported b
 
 ## Creating Shares
 
+{{< hint info >}}
+The below info is outdated as we allow creating federated shares using the graph API. Clients can now discover the available sharing roles and invite federated users using the graph API.
+{{< /hint >}}
```

Comment:
This needs a rework:
- link to libre graph api docs
- hm use existing examples from there?
- prose